lib/models/device.coffee

```diff
@@ -509,6 +544,76 @@ exports.move = (uuid, application, callback) ->
 	.nodeify(callback)
 
 ###*
+# @summary Start application on device
+# @name start
+# @public
+# @function
+# @memberof resin.models.device
+#
+# @param {String} uuid - device uuid
+# @fulfil {String} - application container id
+# @returns {Promise}
+#
+# @example
+# resin.models.device.start('7cf02a6').then(function(containerId) {
+# 	console.log(containerId);
+# });
+#
+# @example
+# resin.models.device.start('7cf02a6', function(error, containerId) {
+# 	if (error) throw error;
+# 	console.log(containerId);
+# });
+###
+exports.start = (uuid, callback) ->
```

Comment:
I see. I like having `Application` as prefix for all functions, including `.restart()`, however that'd involve a breaking change for little benefit. 

I say let's keep them like they are (`.start()` and `.stop()`), but can you make a GitHub issue about renaming all of them, including `.restart()` to include the `Application` suffix?